src/main/java/com/hazelcast/maven/attribution/AggregateAttributionMojo.java

```diff
@@ -19,7 +19,7 @@
  */
 @Mojo(name = "aggregate", defaultPhase = LifecyclePhase.PROCESS_RESOURCES,
         aggregator = true, requiresDependencyCollection = ResolutionScope.COMPILE_PLUS_RUNTIME,
-        requiresDependencyResolution = ResolutionScope.NONE,
+        requiresDependencyResolution = ResolutionScope.TEST,
```

Comment:
Why `TEST` and not `COMPILE_PLUS_RUNTIME` ?

Reply:
> Why `TEST` and not `COMPILE_PLUS_RUNTIME` ?

Short answer - it doesn't work if you don't.

Longer answer - it's failing with compilation errors about being unable to access `test` scoped artefacts.